File: tests/fixtures/sample.java
// Fx.java -- one small class.

package fx;

/** Small arithmetic helpers. */
public final class Fx {

    /** Returns the sum of a and b. */
    public static int add(int a, int b) {
        return a + b;  // a trailing comment
    }

    // `record` is a SOFT keyword: this is an assignment, not a declaration.
    public static final String url = "http://example.com/not-a-comment";
}

File: tests/fixtures/sample.kt
// Fx.kt -- one small file.

package fx

/** Returns the sum of a and b. */
fun add(a: Int, b: Int): Int {
    return a + b  // a trailing comment
}

/** A holder for two numbers. */
data class Pair2(val a: Int, val b: Int)

// `data` alone is a soft keyword and this is an assignment, not a
// declaration -- which is why the row lists `data class`, two words.
val data = listOf(1, 2)

val url = "http://example.com/not-a-comment"
